README.md

```diff
@@ -1,4 +1,5 @@
-[![Anaconda-Server Badge](https://anaconda.org/bsteubing/activity-browser/badges/version.svg)](https://anaconda.org/bsteubing/activity-browser)
+[![conda-forge version](https://img.shields.io/conda/vn/conda-forge/activity-browser.svg)](https://anaconda.org/conda-forge/activity-browser)
+[![conda-forge version](https://img.shields.io/conda/vn/bsteubing/activity-browser.svg)](https://anaconda.org/bsteubing/activity-browser)
```

Comment:
This badge shouldn't be here anymore right?

Reply:
Depends, as @haasad said if you want to maybe show a development version you could keep it.

Reply:
We show the dev version under the [development version header](https://github.com/LCA-ActivityBrowser/activity-browser/blob/40c373ace4a914a863ae62a1ed77826976269581/README.md#development-version) already, but I don't mind if we'd move that to the top of the file 🤷